tests/core/shader.test.js

```diff
@@ -152,6 +152,114 @@ suite('shader', function () {
                                            instance.attributes['src']));
     });
 
+    // Skip since this fails Travis CI Firefox run, even though it works on Chrome, and with local Firefox.
```

Comment:
no, but it is irrelevant since the actual substitution code won't be ever really be fired using Firefox, and I don't have the time to spare to investigate that